src/model.py

```diff
@@ -112,6 +112,14 @@ def initialize(self, args):
         with open(engine_args_filepath) as file:
             vllm_engine_config = json.load(file)
 
+        # Resolve the model path relative to the config file
+        if vllm_engine_config.pop("resolve_model_relative_to_config_file", False):
+            vllm_engine_config["model"] = os.path.abspath(
+                os.path.join(
+                    pb_utils.get_model_dir(), vllm_engine_config["model"]
```

Comment:
would it be possible to constrain this to be a subdirectory of the model_dir root? or does it need to be able to point to files outside of that.

Reply:
This should be possible